.gitignore

```diff
@@ -5,3 +5,4 @@ coverage.out
 coverage.html
 go.work
 go.work.sum
+.gocache
```

Comment:
why do we need to add this to .gitignore? to the best of my knowledge we don't configure the project to use a build cache in the root of the directory so the location of the build cache should be system dependent and vary depending on what you have configured in the `GOCACHE` env var